### DISCUSSION TITLE
position of the element in the source file

### DISCUSSION THREAD

**Discussion by fa66ot** [nlohmann/json#4122] [General]
Hello,
is there a way to get a position of the json element in the source file? Some kind of a metadata call, that would give me a line number of the string in the source file from which this particular json object/element created.
I've checked the desription of json_base_class_t and parse method but was not able to find a mechanism of getting the line number from the source file. 
Any advice?

**Comment by nlohmann** [comment]
No, this is currently not possible.

**Reply by fa66ot** [reply]
ok :(
thx
